components/tag/Tag.js

```diff
@@ -0,0 +1,69 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../box';
+import Button, { IconButton } from '../button';
+import cx from 'classnames';
+import theme from './theme.css';
+import { IconCloseMediumOutline, IconCloseSmallOutline } from '@teamleader/ui-icons';
+
+class Tag extends PureComponent {
+  static propTypes = {
+    className: PropTypes.string,
+    inverse: PropTypes.bool,
+    label: PropTypes.string.isRequired,
+    onLabelClick: PropTypes.func,
+    onRemoveClick: PropTypes.func,
+    size: PropTypes.oneOf(['small', 'medium', 'large']),
+  };
+
+  static defaultProps = {
+    inverse: false,
+    size: 'medium',
+  };
+
+  getCloseButtonColor() {
+    return this.props.inverse ? 'white' : 'neutral';
+  }
+
+  getCloseIcon() {
+    return this.props.size === 'large' ? <IconCloseMediumOutline /> : <IconCloseSmallOutline />;
+  }
+
+  render() {
+    const { className, inverse, label, onLabelClick, onRemoveClick, size, ...others } = this.props;
+
+    const classNames = cx(
+      theme.tag,
```

Comment:
**PP** Use array notation here...

Reply:
I prefer it without the array actually.

Reply:
Nvm, thought you meant `cx([...])`.
Oops 🤓